android/src/main/java/com/twilio/twilio_voice/BackgroundCallJavaActivity.java

```diff
@@ -200,6 +200,106 @@ public void onClick(View v) {
 
                     }
                 });
+                btn1.setOnClickListener(new View.OnClickListener() {
```

Comment:
Do you think we can condense this, a bit too much repetition.

Reply:
Hi, I am very new to native android development, any input from your side will help